README.md

```diff
@@ -131,6 +131,33 @@ For more guidance in running `CRI-O`, visit our [tutorial page](tutorial.md)
 [libpod-hooks]: https://github.com/containers/libpod/blob/v0.6.2/pkg/hooks/README.md
 [spec-hooks]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
 
+#### The HTTP API
+
+CRI-O exposes per default the [gRPC](https://grpc.io/) API to fulfill the
+Container Runtime Interface (CRI) of Kubernetes. Beside this, there exists an
```

Comment:
s/Beside/Besides/

Reply:
Thanks for the review :pray: Adapted as suggested.